shared/vendor/js/jquery.autosuggest.js

```diff
@@ -327,11 +327,7 @@
                         }
                         if(str){
                             if (!opts.matchCase){ str = str.toLowerCase(); }
-                            if(str.search(query) != -1 && values_input.val().search(","+data[num][opts.selectedValuesProp]+",") == -1){
```

Comment:
We shouldn't remove the second part of this clause, as that's the thing that helps avoid showing results that are already in the list.